sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -93,24 +86,10 @@ public EventData(byte[] body, Context context) {
      * @throws NullPointerException if {@code body} is {@code null}.
      */
     public EventData(ByteBuffer body) {
-        this(body, Context.NONE);
-    }
-
-    /**
-     * Creates an event containing the {@code body}.
-     *
-     * @param body The data to set for this event.
-     * @param context A specified key-value pair of type {@link Context}.
-     * @throws NullPointerException if {@code body} or if {@code context} is {@code null}.
-     */
-    public EventData(ByteBuffer body, Context context) {
-        Objects.requireNonNull(body, "'body' cannot be null.");
-        Objects.requireNonNull(body, "'context' cannot be null.");
-
-        this.body = body;
+        this.body = Objects.requireNonNull(body, "'body' cannot be null.").array();
```

Comment:
Should this call into the `byte[]` constructor instead?

```java
this(Objects.requireNonNull(body, "'body' cannot be null.").array());
```

Reply:
good catch. Fixed